platform/ios/MGLMultiPoint.mm

```diff
@@ -0,0 +1,103 @@
+#import "MGLMultiPoint.h"
+#import "MGLTypes.h"
+
+#import <mbgl/util/geo.hpp>
+
+@implementation MGLMultiPoint
+{
+    CLLocationCoordinate2D *_coords;
+    size_t _count;
+    mbgl::LatLngBounds _bounds;
+}
+
+- (instancetype)initWithCoordinates:(CLLocationCoordinate2D *)coords
+                              count:(NSUInteger)count
+{
+    self = [super init];
+
+    if (self)
+    {
+        _count = count;
+        _coords = (CLLocationCoordinate2D*)malloc(_count * sizeof(CLLocationCoordinate2D));
+
+        for (NSUInteger i = 0; i < _count; i++)
+        {
+            _coords[i] = coords[i];
+            _bounds.extend(mbgl::LatLng(coords[i].latitude, coords[i].longitude));
+        }
+    }
+
+    return self;
+}
+
+- (void)dealloc
+{
+    free(_coords);
+}
+
+- (CLLocationCoordinate2D)coordinate
+{
+    if ([self isMemberOfClass:[MGLMultiPoint class]])
+    {
+        [[NSException exceptionWithName:@"MGLAbstractClassException"
+                                 reason:@"MGLMultiPoint is an abstract class"
+                               userInfo:nil] raise];
```

Comment:
Yeah, this would be good, but it's slightly more than trivial since it would be called from several class implementations. @1ec5 Could you take a look at this? It's not a blocker, so it could be b2 or b3.